core/services/ocr2/plugins/llo/config/config.go

```diff
@@ -97,8 +97,8 @@ func (p PluginConfig) Validate() (merr error) {
 		merr = errors.Join(merr, errors.New("llo: DonID must be specified and not zero"))
 	}
 
-	if len(p.Servers) == 0 {
-		merr = errors.Join(merr, errors.New("llo: At least one Mercury server must be specified"))
+	if len(p.Servers) == 0 && len(p.Transmitters) == 0 {
```

Comment:
I'd be surprised if this passes tests.

I think you will need a bit more than this.